### ISSUE TITLE
Bug in restoring session with --closeAllOpenWindows from Terminal

### ISSUE THREAD

**Issue by dadwalgd** [johannesjo/linux-window-session-manager#91]
## Your Environment

* Version used: 
* Operating System and version: 
* Desktop Environment: 
* Node version: 
* List of Applications that are open (lwsm save) or should be opened (lwsm restore): 

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

**Comment by johannesjo** [comment]
How about executing `lwsm restore &`?

Apart from that: PRs are welcome!

**Comment by dadwalgd** [comment]
> How about executing `lwsm restore &`?
> 
> Apart from that: PRs are welcome!

No, that doesn't work. It closes the terminal (during closeAllWindows process) and then nothing happens(no restoration).
